geometry_msgs/PoseArray cluster_poses
---
geometry_msgs/PoseArray poses